dffml/util/cli/cmd.py

```diff
@@ -87,21 +89,32 @@ def add_subs(self, add_from: "CMD"):
                 except argparse.ArgumentError as error:
                     raise Exception(repr(add_from)) from error
 
+        for field in dataclasses.fields(add_from.CONFIG):
+            arg = mkarg(field)
+            if isinstance(arg, Arg):
+                try:
+                    # Add condition for positional arguments
+                    self.add_argument("-" + field.name, **arg)
+                except argparse.ArgumentError as error:
+                    raise Exception(repr(add_from)) from error
+
+
+@config
+class CMDConfig:
+    log: str = field(
```

Comment:
```suggestion
    log: int = field(
```

Reply:
We enter a string value in CLI for `-log` which gets changed to it's corresponding int value [here](https://github.com/intel/dffml/blob/master/dffml/util/cli/cmd.py#L26). Providing an int value doesn't work in this case.

Reply:
Sounds good!